### ISSUE TITLE
unable to navigate drop-down lists on webpages

### ISSUE THREAD

**Issue by abdullah5490** [nvaccess/nvda#5671]
hi developers. i want to report an issue that i'm facing with NVDA's latest release. the drop-down list boxes are not being read by NVDA. for example, on standard facebook site, when we mention someone by typing his./her name after "@" symbol, a new menu appears reflecting the results of the typed name. NVDA cannot render that list with the aerrow keys while remaining the correct person. as a result of this, i am unable to mention or tag someone, select the group/page name while sharing a post from somewhere to somewehere else, or perform search on facebook to find a friend. because instant results are not accessible with NVDA. this problem is affecting firefox, chrome, edge, and internet explorer as well. previously, it was working fine and there is no difficulty in tagging or finding someone's name on facebook for whatever purposes. another thing, i've scene on a friend's computer, this feature works fine with another screenreader (JAWS). please take notice of it, or guide me if anything that i need to set-up for getting it in to work. thanks for reading.

**Comment by jcsteh** [comment]
This works as expected for me in Firefox. Please provide thorough 
details, including exactly what you did, what you expected and what 
happened instead (e.g. what was spoken). Also, please try with any 
add-ons disabled, as add-ons might be interfering with NVDA's normal 
functionality.

**Comment by abdullah5490** [comment]
let's take one example. i am trying to mention one of my friend in my comment on a story on facebook. let suppose, i want to mention "john", where John is the name of the targetted person. i typed @john and result matching the "john" input displayed in a drop-down list on the screen, which need to be select the correct person with aerrow keys, followed by enter key. but NVDA does not read that newly shown list, does not read the names of the resulted persons with the press of down or up aerrows. but if i press enter blindly after pressing few downs, a person got selected in the comment text. which means, the list is being navigated with the aerrows, but NVDA does not telling what is under the curser, and when i press enter, the person get marked on which the curser was standing at that moment. it makes unable to mention anyone in a comment or a new post. same thing happening while tagging a friend in a post, selecting sharing location of a post, using facebook search box, and many others where instant results are been showed. i have no add-on installed in firefox other than IDM extention, and none of the add-on is present in NVDA other than eloquence and fake clipboard announcement.

**Comment by jcsteh** [comment]
Okay. As I said, this works as expected for me in Firefox. What version of Firefox are you using? What does NVDA report if you press NVDA+tab after pressing down arrow in one of these lists?

**Comment by abdullah5490** [comment]
When I press NVDA+tab, it said, “comment edit combo box”. And its firefox 43.0.2, the problem was also happening in just previous version of FF.

 

Regards,

Abdullah Zafar.

Coordinator training and audio department at Masiha educational and welfare society (MASEWS).

Moderator and writer at united inclusive planet technology lovers.

Skype:  <mailto:abdullah.93@outlook.com> abdullah.93@outlook.com

Facebook:  <http://www.facebook.com/prince.abdullah93> my profile

Twitter:  <http://www.twitter.com/abdullah5490> abdullah5490 on twitter

Sent from outlook 2016 e-mail client

**Comment by abdullah5490** [comment]
still waiting for reply, or assigning.

**Comment by ehollig** [comment]
This works as expected for me in Firefox 54.0.1. @abdullah5490 is this still happening with the latest version of Firefox and latest NVDA? You may also want to try this with addons disabled in Firefox, in case one may be causing issues.

**Comment by abdullah5490** [comment]
@ehollig it sometime works, and sometime not. even I don't have any add-on enabled

**Comment by ehollig** [comment]
Okay, are you using the latest version of Firefox? As I said, this works as expected for me in Firefox. Maybe a NVDA log would be useful that shows this behavior.

**Comment by ehollig** [comment]
I am closing this as worksforme, as no one is able to reproduce the issue.
If I go to the Facebook site and type the @ symbol in the edit field for "What's on your mind?", NVDA does not say "List" until I type two or three letters. After NVDA reports "List", I can use the down arrow to select a name or page to mention by pressing the enter key.
